### PR TITLE
fix client.py to work with python3.4+

### PR DESCRIPTION
**Pull request by rgerganov**

### CODE REVIEW

**Comment by cemeyer** [comment]
Oh yeah, sorry.  I have some of these exact same changes in my local tree but didn't clean them up and push them yet.

File: client.py [hunk 1]
```diff
@@ -51,11 +51,11 @@ def parse_ServerResponse(self, response):
         assert x.tag == '{http://schemas.xmlsoap.org/soap/envelope/}Envelope'
         r = x.find('.//{http://ctkipservice.rsasecurity.com}' + outer)
         ad = r.find('{http://ctkipservice.rsasecurity.com}AuthData')
-        assert ad.text == self.auth == response.headers.get('Authorization')
+        #assert ad.text == self.auth == response.headers.get('Authorization')
```
Comment (cemeyer) [inline client.py:54]: My server doesn't send `Authorization` either, but for me the `ad.text == self.auth` assertion still held.  Does it not for you?

File: client.py [hunk 2]
```diff
@@ -122,17 +122,24 @@ def serverFinished(self, session_id, server_nonce, client_none=None):
         if client_none is None:
             client_nonce = random.getrandbits(16*8)
         cipher = PKCS1_OAEP.new(self.server_pubkey)
+        client_nonce = client_nonce.to_bytes(16, byteorder='big')
```
Comment (cemeyer) [inline client.py:125]: Oh, that's a lot better than what I have (`bytes(bytearray([random.getrandbits(8) for i in range(16)]))`).

**Review by cemeyer — COMMENTED** [review]
Mostly LGTM.  I found all of these changes to be necessary too but hadn't pushed them yet, sorry.  I think the assertion change can be reduced a little, if you can confirm it wasn't needed for you.

**Comment by cemeyer** [comment]
FYI: basically same as https://github.com/dlenski/rsa_ct_kip/pull/1

**Comment by rgerganov** [comment]
Not needed anymore.

**Pull request closed without merging**